src/app/component/general/eco-news/create-news/create-news-interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface CreateNewsInterface {
```

Comment:
Please rename it without using the verb in the interface name

Reply:
fixed